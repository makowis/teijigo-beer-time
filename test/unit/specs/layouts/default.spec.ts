import { mount, RouterLinkStub } from '@vue/test-utils';
import { Component } from 'vue';
import DefaultLayout from '@/layouts/default.vue';

describe('index.vue', () => {
  const NuxtMock:Component  = {
    template: '<div>Nuxt</div>'
  }
  const vm = mount(DefaultLayout, {
    stubs: {
      NuxtLink: RouterLinkStub,
      Nuxt: NuxtMock,
    }
  }).vm;  

  it('should match the snapshot', () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
