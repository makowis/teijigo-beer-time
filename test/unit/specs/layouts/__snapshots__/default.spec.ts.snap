// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue should match the snapshot 1`] = `
<div
  id="app"
>
  <nav
    class="menuContainer"
    id="menu-container"
  >
    <ul
      class="menu"
    >
      <li
        class="menuItem"
      >
        <a>
           トップ 
        </a>
      </li>
       
      <li
        class="menuItem"
      >
        <a>
           DISCOGRAPHY 
        </a>
      </li>
       
      <li
        class="menuItem"
      >
        <a>
           メンバー紹介 
        </a>
      </li>
       
      <li
        class="menuItem"
      >
        <a>
           びーにゃの部屋 
        </a>
      </li>
       
      <li
        class="menuItem"
      >
        <a
          href="https://teijigo.booth.pm/items/1093403"
          target="_blank"
        >
          
        通販(BOOTH)
      
        </a>
      </li>
    </ul>
  </nav>
   
  <div
    class="mainContainer"
    id="main-container"
  >
    <a>
      <img
        class="logo"
        src="@/assets/logo_1.png"
      />
    </a>
     
    <div>
      Nuxt
    </div>
     
    <footer
      class="footer"
    >
      <p>
        ©定時後ビールタイム
      </p>
    </footer>
  </div>
</div>
`;
